src/sider/Side.tsx

```diff
@@ -53,14 +47,19 @@ const Side = ({ tittel, aktivtMenypunkt, children }: SideProps) => {
           {isPride() && <Pride>&nbsp;</Pride>}
           <Personkort />
         </div>
-        <AdaptableRow>
-          <nav className="min-w-[15rem] mr-2">
+        <div className={"flex max-[700px]:flex-wrap"}>
+          <nav className="max-[700px]:w-full min-w-[15rem] min-[700px]:mr-2">
             <GlobalNavigasjon aktivtMenypunkt={aktivtMenypunkt} />
             {isEaster() && <Easter />}
-            {toggles.isHuskelappEnabled && <OpenHuskelappModalButton />}
+            {toggles.isHuskelappEnabled &&
+              (isExistingHuskelapp ? (
```

Comment:
Kunne evt gjort dette i en egen komponent så man unngår kall til huskelapp-api når toggle er disabled.

Reply:
Flyttet det til `Huskelapp.tsx` 😊
Snakket litt med Stine om at "trenger oppfølging" kanskje ikke er det beste ordet for å beskrive funksjonaliteten, så synes vi bør vente litt med full renaming overalt 😊